app/controllers/manual_sections_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ManualSectionsController < ApplicationController
+  def show
+    @section = Section.find(content_id: params[:content_id])
```

Comment:
@h-lame I'm going add some 404 errors to the App this morning. Master redirects to `manuals_path` and displays `flash[:danger]`. Is that enough for this or should I be doing `status: 404`, raising `ActiveRecord::RecordNotFound` or raising `ActionController::RoutingError.new('Not Found')`?